README.md

```diff
@@ -0,0 +1,94 @@
+libgit4cpp
+==========
+
+libgit4cpp is a lightweight C++ wrapper around the [libgit2](https://libgit2.org/) C
+library. It was mainly developed for use in
+[taskolib](https://taskolib.github.io/taskolib/) and covers only a tiny subset of the
+functionality of the original library.
+
+The library API is documented with Doxygen on the web page
+https://taskolib.github.io/libgit4cpp/ .
```

Comment:
The page does not exist

Reply:
I think it will come into existence once the Github actions in this PR are active. I'll merge and we'll have a look.